proto/storage/blob.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+option go_package = "storage";
+option java_package = "io.stackrox.proto.storage";
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+package storage;
+
+// Next Tag: 6
+message Blob {
```

Comment:
Should we add a size field?  Reason being for BYODB and ACSCS we are not going to be able to determine how much space we are using for these.  If we track the size we can dump it as a metric from time to time.